blocks/bricks/cost.py

```diff
@@ -70,3 +71,12 @@ def apply(self, y, y_hat):
         return (
             tensor.sum(tensor.neq(y, y_hat.argmax(axis=1))).astype(floatX) /
             y.shape[0].astype(floatX))
+
+
+class BinaryMisclassificationRate(Cost):
+    @application(outputs=["error_rate"])
+    def apply(self, y, y_hat):
+        # Here we have to cast both operands to floatX explicitly.
+        # Because int64 / float32 = float64 in Theano, unfortunately.
+        return (tensor.sum(tensor.neq(y, y_hat > 0.5)).astype(floatX) /
```

Comment:
Good point. I do not think there is much sense in having them as separate bricks. @bartvm , @dwf , what do you think?